requirements.txt

```diff
@@ -1,3 +1,4 @@
+moviepy
```

Comment:
I've moved those dependencies around between setup and even tox because of your suggestion below but it kept giving me errors. I'm sure something can be streamlined here, but I could not make it work the way you suggested (which would have been the most clean way, I agree)